commit eac62497aec204568a494743f829d922787d69c5
Author: Curtis Vogt <curtis.vogt@gmail.com>
Date:   Thu Sep 21 15:51:52 2017 -0500

    Ask for credentials again when passphrase is wrong
    
    When trying to decode the private key it looks like LibSSH2 returns a
    LIBSSH2_ERROR_PUBLICKEY_UNVERIFIED when the passphrase is incorrect.

diff --git a/src/transports/ssh.c b/src/transports/ssh.c
index 172ef413c..ec3b0b6ff 100644
--- a/src/transports/ssh.c
+++ b/src/transports/ssh.c
@@ -420,8 +420,8 @@ static int _git_ssh_authenticate_session(
 		}
 	} while (LIBSSH2_ERROR_EAGAIN == rc || LIBSSH2_ERROR_TIMEOUT == rc);
 
-        if (rc == LIBSSH2_ERROR_PASSWORD_EXPIRED || rc == LIBSSH2_ERROR_AUTHENTICATION_FAILED)
-                return GIT_EAUTH;
+	if (rc == LIBSSH2_ERROR_PASSWORD_EXPIRED || rc == LIBSSH2_ERROR_AUTHENTICATION_FAILED || rc == LIBSSH2_ERROR_PUBLICKEY_UNVERIFIED)
+		return GIT_EAUTH;
 
 	if (rc != LIBSSH2_ERROR_NONE) {
 		if (!giterr_last())
